docs/vendor-requirements.md

```diff
@@ -234,7 +234,8 @@ Example structure in device managed object using the inventory API:
 ## External ID
 
 
-The External ID is displayed by the UI in the tab "Identity". The fragments `externalId` and `type` must be present in the managed object of the device using the identity API. NOTE: The externalID is not stored in the device managed object using the inventory API.
+The External ID is displayed by the UI in the tab "Identity". The fragments `externalId` and `type` must be present in the managed object of the device using the identity API. 
+NOTE: The externalID is not stored in the device managed object using the inventory API.
 
```

Comment:
NOTE: The externalID is not stored in the device managed object using the inventory API. 
is richtig aber dann solltest du besser erklären bzw die Indentity API erwähnen https://cumulocity.com/api/10.11.0/#tag/External-IDs